Week01/Problem04/Abhiraj_4.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/local/bin/python3.6
+
+def palindrome(n):
```

Comment:
If an integer is *cast* as a string, i.e., 
```
>>> a = 52
>>> str(a)
'52'
```
then you can use list slicing to determine if the number is a palindrome:
```
def palindrome_test(value):
    return(str(value) == str(value)[::-1])
```
which will return `True` if the number `value` is a palindrome, and `False` otherwise.